rzup/Cargo.toml

```diff
@@ -10,25 +10,37 @@ repository = { workspace = true }
 [package.metadata.release]
 release = false
 
+[[bin]]
+name = "rzup"
+path = "src/bin/rzup.rs"
+
+[features]
+default = ["cli", "install"]
+cli = ["dep:clap", "dep:indicatif", "dep:colored", "dep:is-terminal"]
+install = ["dep:reqwest", "dep:tar", "dep:xz", "dep:flate2", "dep:fs2"]
+
```

Comment:
What is the intention with these feature flags?

Reply:
They are intended to support both library usage where installation and cli features aren't necessary - such as in #2752, and the rzup binary. 

We could drop the cli feature since it provides little benefit - library users will end up disabling both features, or just enabling install.